tests/functional/object/mcg/test_mcg_replication_target_unreachable.py

```
@@ -264,3 +264,133 @@ def test_noobaa_replication_target_unreachable_iam_key_revocation(
+            pattern="pre-disrupt-",
+            mcg_obj=mcg_obj,
+        )
+        assert compare_bucket_object_list(
+            mcg_obj, source_bucket.name, target_bucket.name, timeout=600
+        ), "Replication did not work before underlying bucket deletion"
+        logger.info("Initial replication verified successfully")
+
+        # 4. Delete the underlying MCG bucket of the target's store
+        mcg_obj.s3_resource.Bucket(underlying_bucket_name).objects.all().delete()
+        mcg_obj.s3_resource.Bucket(underlying_bucket_name).delete()
+        logger.info(f"Underlying MCG bucket deleted: {underlying_bucket_name}")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)